zerver/webhooks/github/tests.py

```diff
@@ -179,18 +179,18 @@ def test_issue_unassigned_with_custom_topic_in_url(self) -> None:
 
     def test_issue_labeled(self) -> None:
         expected_topic = "testing-gh / issue #9 idk man"
-        expected_message = "sbansal1999 labeled [issue #9](https://github.com/sbansal1999/testing-gh/issues/9):\n\n~~~ quote\nThis is some random Issue description which will be used to test the Zulip GitHub Integration.\n~~~"
+        expected_message = "[Issue #9](https://github.com/sbansal1999/testing-gh/issues/9) was labeled bug by [sbansal1999](https://github.com/sbansal1999)."
         self.check_webhook("issues__labeled", expected_topic, expected_message)
 
     def test_issue_labeled_with_custom_topic_in_url(self) -> None:
         self.url = self.build_webhook_url(topic="notifications")
         expected_topic = "notifications"
-        expected_message = "sbansal1999 labeled [issue #9 idk man](https://github.com/sbansal1999/testing-gh/issues/9):\n\n~~~ quote\nThis is some random Issue description which will be used to test the Zulip GitHub Integration.\n~~~"
+        expected_message = "[Issue #9 idk man](https://github.com/sbansal1999/testing-gh/issues/9) was labeled bug by [sbansal1999](https://github.com/sbansal1999)."
         self.check_webhook("issues__labeled", expected_topic, expected_message)
 
     def test_issue_unlabeled(self) -> None:
         expected_topic = "testing-gh / issue #9 idk man"
-        expected_message = "sbansal1999 unlabeled [issue #9](https://github.com/sbansal1999/testing-gh/issues/9):\n\n~~~ quote\nThis is some random Issue description which will be used to test the Zulip GitHub Integration.\n~~~"
+        expected_message = "[Issue #9](https://github.com/sbansal1999/testing-gh/issues/9) was unlabeled bug by [sbansal1999](https://github.com/sbansal1999)."
```

Comment:
Can you post this string in chat.zulip.org for feedback? I feel like the English feels awkward; best might be `{name} added the [labe_name]{label_url}` label to [Issue #9]{).` or something like it.